src/clj/rems/multipart.clj

```diff
@@ -41,6 +42,56 @@
     (testing "too large, byte at a time"
       (is (= [:too-large [0 1 2]] (copy {:buffer-size 1 :max-size 3}))))))
 
+(defn scan-for-malware
+  "Feeds byte-array to STDIN and runs executable at malware-scanner-path returns true if malware executable returns a non-zero status-code, false otherwise, logs STERR of executable"
+  [malware-scanner-path byte-array]
+  (let [scan-output (sh "sh" "-c" malware-scanner-path :in byte-array)]
+    {:detected (not= (:exit scan-output) 0)
+     :out      (:out scan-output)
+     :err      (:err scan-output)}))
```

Comment:
These days we use the map format that does not try to indent the keys and values, i.e.:
```clj
    {:detected (not= (:exit scan-output) 0)
     :out (:out scan-output)
     :err (:err scan-output)}))
     ```

Reply:
That's unexpected. 

When I run `lein cljfmt fix` that does not happen (it uses the configs specified in `project.clj`). Perhaps the difference is in how you run `cljfmt`. We for example do not use the `.cljfmt.edn` file because the required config is already in `project.clj`.

Reply:
Hmm, I get the following:

```
$ lein cljfmt check src/clj/rems/multipart.clj
All source files formatted correctly
```

Though it was originally formatted using `cljfmt` v. 0.11.2 pointing to this configuration:

```
{:paths ["project.clj" "src/clj" "src/cljc" "src/cljs" "test/clj" "test/cljc" "test/cljs"] ; need explicit paths to include cljs
 :legacy/merge-indents? true
 :indents {delay [[:inner 0]]}
 :remove-consecutive-blank-lines? false}
```

With ` :legacy/merge-indents? true` stemming from [breaking changes in v0.11](https://github.com/weavejester/cljfmt/blob/master/README.md#breaking-changes-in-011x)

Reply:
`$ lein cljfmt fix src/clj/rems/multipart.clj` does not change anything either.

Reply:
Perhaps it originally came from your editor settings? I guess `cljfmt` doesn't actually consider this case at all (it's OK either way).